File: Examples/Effects/Demo/ViewController.m

#import "ViewController.h"
#import "MyScene.h"

@interface ViewController ()
@property (nonatomic, weak) IBOutlet SKView *skView;
@end

@implementation ViewController

- (void)viewDidLoad {
  [super viewDidLoad];

//  self.skView.showsFPS = YES;
//  self.skView.showsNodeCount = YES;
//  self.skView.showsDrawCount = YES;
}

- (BOOL)shouldAutorotate {
  return YES;
}

- (BOOL)prefersStatusBarHidden {
  return YES;
}

- (NSUInteger)supportedInterfaceOrientations {
  return UIInterfaceOrientationMaskLandscape;
}

- (void)viewWillLayoutSubviews {
  [super viewWillLayoutSubviews];

  if (self.skView.scene == nil) {
    SKScene *scene = [MyScene sceneWithSize:self.skView.bounds.size];
    [self.skView presentScene:scene];
  }
}

@end

File: Examples/Effects/Demo/ViewController.swift

import SpriteKit

class ViewController: UIViewController {
  @IBOutlet weak var skView: SKView!
  
  override func viewDidLoad() {
    super.viewDidLoad()
    
    skView.showsFPS = true
    skView.showsNodeCount = true
    skView.showsDrawCount = true
  }

  override func viewWillLayoutSubviews()  {
    super.viewWillLayoutSubviews()
    
    if skView.scene == nil {
      let scene = MyScene(size: skView.bounds.size)
      skView.presentScene(scene)
    }
  }

  override func prefersStatusBarHidden() -> Bool {
    return true
  }

  override func shouldAutorotate() -> Bool {
    return true
  }

  override func supportedInterfaceOrientations() -> UIInterfaceOrientationMask {
    return UIInterfaceOrientationMask.Landscape
  }
}
